rules/collections/default/kotlin.yaml

```diff
@@ -4,3 +4,9 @@ collections:
     patterns:
       - "(?i)io.ktor.server.routing.(post|get|delete|put).*"
     tags:
+    
+  - id: Collections.Spark.HttpFramework
+    name: Spark Http microframework
+    patterns:
+      - "(?i)spark.Spark.*(post|get|delete|put|options|halt).*"
+    tags:
```

Comment:
@HiteshMah-Jan We already have a rule for Spark in java.yaml file, that will be used for kotlin as well . Refer - https://github.com/Privado-Inc/privado/blob/dev/rules/collections/default/java.yaml